Fix empty find what

spereverde opened the pull request:
- fix error when leaving the 'find what' field empty (see https://github.com/collective/collective.searchandreplace/issues/43) 
- when leaving the 'find what' field empty, it will now show the validation error (red border and mention field is required) on the field which it did not before
- no test added yet, still have to figure out where best to test triggering the validation

- note: this was based on the cleanup branch (PR https://github.com/collective/collective.searchandreplace/pull/44 )

mauritsvanrees commented:
I tested it, and the last commit is okay. But this PR still contains the commits from the merged PR #44. It should be rebased on master. Seeing the force-push two days ago it seems like you might have tried already, but I guess something went wrong.

Also, I would suggest creating PR branches directly in the collective instead of in an own fork. But if that does not suit your workflow, a fork is fine.

spereverde commented:
I force pushed before because all the PR's were based on the cleanup branch, I rebased them from that branch before.
I now rebased all the branches from the collective master.
Next time I will make the PR's  on the collective, that will indeed make things a bit easier :-)

mauritsvanrees reviewed (approved):
Thanks!
